RESTserver/pom.xml

```diff
@@ -13,45 +13,81 @@
     <packaging>war</packaging>
 
     <dependencies>
-
         <dependency>
             <groupId>com.softserve.if072</groupId>
             <artifactId>common-modules</artifactId>
-            <version>1.0-SNAPSHOT</version>
         </dependency>
 
-        <!-- https://mvnrepository.com/artifact/org.mybatis/mybatis -->
         <dependency>
-            <groupId>org.mybatis</groupId>
-            <artifactId>mybatis</artifactId>
-            <version>3.4.1</version>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
         </dependency>
 
         <dependency>
-            <groupId>org.mybatis</groupId>
-            <artifactId>mybatis-spring</artifactId>
-            <version>1.3.0</version>
+            <groupId>log4j</groupId>
+            <artifactId>log4j</artifactId>
```

Comment:
Since I've mentioned all dependencies within <dependencyManagement> section in parent pom.xml file, all children pom.xml files don't inherit them as a default. You should explicitly specifying the required dependencies in children pom.xml files (you can also omit the version information which is specified in parent pom.xlm), otherwise modules won't compile.